kontrakter/felles/lukkSak/arbeidsversjon/lukkSak.schema.json

```diff
@@ -0,0 +1,144 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://domstol.no/lukkSak/arbeidsversjon/lukkSak",
```

Comment:
vi kjører forskjellig $id, kan vi lage det likt?
Her har jeg ville sagt: "$id": "https://domstol.no/felles/arbeidsversjon/lukkSak" se  forøvrig:
begjæring dom: "https://politiet.no/dom/1.0/begjaeringDom"
Dom: "$id": "https://domstol.no/dom/domsavjoerelse/arbeidsversjon/Domsavgjoerelse.schema.json"
kjennelseVaretekt: "$id": "https://domstol.no/da/varetekt/1.2/KjennelseVaretekt.schema.json"

Retningslinjer på Confluence er litt missvisende fordi da hadde vi en katalog per aktør ikke per funksjonsområde som vi har nå.
Jeg lager en RFC med et forslag, som vi kan bruke til å diskutere slik at vi får det på samme format nå på de nye meldingene.